frontend/src/pages/EditTeacher.vue

```diff
@@ -170,33 +170,25 @@ export default {
 
   data() {
     return {
-      name: "",
-      PhoneNumber: "",
-      Email: "",
-      source: "",
-      EnglishProficiency: "",
-      SecondLanguageProficiency: "",
-      SecondLanguage: "",
-      Notes: "",
-      selected: {
-        NativeLanguage: "",
-        SecondLanguage: "",
-      },
-      nativeLanguage: "",
       languages: [],
       file: null,
+      teacherData: {
+        nativeLanguage: { NativeLanguage: '' },
+        secondLanguage: {NativeLanguage: ''}
+      }
+
     };
   },
   computed: {
-    ...mapGetters(["API_nativeLanguage"]),
-
+    ...mapGetters(["API_nativeLanguage", "getTeacherByTeacherId", "getAllTeachers"]),
```

Comment:
Do we need to map the getAllTeachers getter? I don’t see the Teachers list data being used, and don’t think it’s needed since we have the ability to get a single teacher with getTeacherByTeacherId.